docs/content/en/release-notes/_index.md

```diff
@@ -5,37 +5,52 @@ weight: 2
 
-**Latest release:** K8ssandra 1.4.0
+K8ssandra Operator 1.0.0 is a significant advancement of the overall K8ssandra project. K8ssandra Operator adds:
 
-Release date: 19-November-2021
+* Multi-cluster support for Kubernetes
+* Native support for multi-region/multi-DC Cassandra databases in Kubernetes
+
+As a platform architect or developer, you can declaratively manage the cluster topology and map Cassandra racks to cloud availability zones or failure domains.  
+
+K8ssandra Operator also adds a new observability service, which allows you to choose and deploy your preferred metrics collection and visualization tools. That is, with K8ssandra Operator 1.0.0, you are not limited to using only the Prometheus and Grafana deployed by K8ssandra 1.4.x. 
+
+**Latest releases:** 
+
+* K8ssandra Operator 1.0.0 on 03-March-2022
+* K8ssandra 1.4.1 on 02-December-2021
 
 {{% alert title="Note" color="success" %}}
-**K8ssandra 1.4.0** implements a number of changes, enhancements, and bug fixes. This topic summarizes the key revisions in 1.4.0 and prior 1.x releases, provides links to the associated issues in our GitHub repo, and contains important upgrade considerations.
+K8ssandra Operator 1.0.0 and K8ssandra 1.4.1 implement a number of changes, enhancements, and bug fixes. This topic summarizes the key revisions, provides links to the associated issues in our GitHub repo, and contains important upgrade considerations.
+
+In addition to this documentation, see the open-source GitHub repos:
+
+* K8ssandra Operator: https://github.com/k8ssandra/k8ssandra-operator
+* K8ssandra: https://github.com/k8ssandra/k8ssandra
 
 **Reminder**: We've migrated the cass-operator GitHub repo from https://github.com/datastax/cass-operator to https://github.com/k8ssandra/cass-operator. Refer to the new repo for the latest Cass Operator developments.
 {{% /alert %}}
 
 ## Prerequisites
 
-* A Kubernetes environment from v1.17 (minimum supported) up to v1.22 (current tested upper bound) - local or via a supported cloud provider
-* [Helm](https://helm.sh/) v3. Note that K8ssandra 1.4.0 has been tested with Helm v3.5.3. Recommendation: specifically avoid Helm 3.6.0 and 3.6.1 due to a known CVE and subsequent regression. Our testing will resume with Helm v3.6.2 and later. Related: see GH issue [1103](https://github.com/k8ssandra/k8ssandra/issues/1103).
+* A Kubernetes environment from v1.19 (minimum supported) up to v1.22 (current tested upper bound) - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3. Note that K8ssandra 1.4.x and K8ssandra Operator have been tested with Helm v3.5.3. Recommendation: specifically avoid Helm 3.6.0 and 3.6.1 due to a known CVE and subsequent regression. Our testing may resume with Helm v3.6.2 and later. Related: see GH issue [1103](https://github.com/k8ssandra/k8ssandra/issues/1103).
```

Comment:
We don't really matrix test for helm versions since it's a client side too, but anecdotally we may want to remove this stuff, I checked my system and I'm using:

```
version.BuildInfo{Version:"v3.8.0", GitCommit:"d14138609b01886f544b2025f5000351c9eb092e", GitTreeState:"clean", GoVersion:"go1.17.6"}
```

On one system and then

```
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
```

On another, both work as expected.

Reply:
Updated in next commit.